### DISCUSSION TITLE
Where are boundaries sourced from?

### DISCUSSION THREAD

**Discussion by a-random-lemurian** [osm-americana/openstreetmap-americana#926] [Q&A] [answered]
Where does Americana get its boundaries from? I'm trying to adapt the style to a fictional map called OpenGeofiction, and maritime boundaries are showing up inconsistently, and if they do, the boundary labels are real-world countries as opposed to the ones on the fictional map.

**Comment by ZeLonewolf** [comment, marked as answer]
Boundaries come from OSM at all zooms. I use the planetiler argument `--boundary-osm-only` to enable this instead of using Natural Earth boundaries.

The tile generation script is here: https://github.com/ZeLonewolf/planetiler-scripts/blob/main/local/render_once.sh

**Reply by a-random-lemurian** [reply]
Is there a way to ensure that boundaries are coming from OpenGeofiction's database instead of OSM? Do I really need an S3 bucket?

**Reply by ZeLonewolf** [reply]
"From OSM" just means "from the supplied .osm.pbf file"

**Reply by a-random-lemurian** [reply]
Okay, so I just have to run that script on an ogf-planet.osm.pbf download of OpenGeofiction and somehow find a way to host the generated tiles?

**Reply by ZeLonewolf** [reply]
That's glossing over a lot of the details but essentially yes. You can read more about my approach here: https://www.openstreetmap.org/user/ZeLonewolf/diary/402227
